test/misc.jl

```diff
@@ -286,7 +286,7 @@ end
 # timing macros
 
 # test that they don't introduce global vars
-global v11801, t11801, names_before_timing
+global v11801::Any, t11801::Any, names_before_timing::Any
```

Comment:
```suggestion
global v11801, t11801, names_before_timing
```
Type declarations being required seems breaking?

Reply:
It's required in the full thing, I believe I changed it back for this PR. I can undo the test changes.